flake8_html/plugin.py

```diff
@@ -142,14 +156,33 @@ def finished(self, filename):
                 unique_messages > 1 or any(e[2] > 1 for e in errs),
                 errs
             ))
+            if self.pep8report:
+                # pep8 report - Step 1: gather errors
+                for err in errors:
+                    pep8_report_errs.append(
+                        (
+                            err.filename, err.line_number, err.column_number,
+                            err.code, err.text
+                        )
+                    )
+
+        if self.pep8report:
+            # pep8 report - Step 2: sort gathered errors by
+            # filename, line- and column number
+            pep8_report_errs.sort(key=lambda err: (err[0], err[1], err[2]))
```

Comment:
...then the key function here is the much more readable

    lambda err: (err.filename, err.line_number, err.column_number)